build-tools/packages/build-cli/src/commands/merge/branches.ts

```diff
@@ -62,6 +63,19 @@ export default class MergeBranch extends BaseCommand<typeof MergeBranch> {
 			default: true,
 			allowNo: true,
 		}),
+		checkPr: Flags.boolean({
+			description: "Use --no-checkPr to skip checking for a PR. Useful for testing.",
+			hidden: true,
+			default: true,
+			allowNo: true,
+		}),
+		cleanup: Flags.boolean({
+			description:
+				"Use --no-cleanup to skip cleanup of branches if there's a failure. Useful for testing.",
+			hidden: true,
```

Comment:
do we plan on removing cleanup flag once the command is fully functional?

Reply:
Probably not -- it's hidden, so unless you read the source or know it's there, it's unlikely to trip anyone up.